components/articles/ArticlePreview.js

```diff
@@ -24,7 +24,7 @@ const ArticlePreview = ({
         <figure className={classNames('image', imageClassName)}>
           <ArticleLink slug={slug}>
             <a>
-              <img alt={title} src={imagePath || mockImagePath} />
+              <img alt={title} src={imageUrl || mockImagePath} />
```

Comment:
please change `const mockImagePath = '/static/images/photo5.jpg';`, cause on case not root route it will be resolved as `/some/path/static...`